docs/application/dotnet/guides/applications/service_application.md

```diff
@@ -0,0 +1,213 @@
+This approach allows reusing modules between different applications such as cyclist applications, automotive solutions, and sport activity apps. The core module, with business logic, works in the background and performs the task it was designed for until it gets closed by system events like `OnLowMemory` or `OnLowBattery`. The activity of the core module remains unaffected by any user action such as accidental closing of the app UI.
+
+The main Service Application API features include: 
+-  Application states:
+
+    A Tizen service application has different states which it transitions through during its life-cycle.
+
+- Event callbacks:
+  
+    The service application can receive both basic system events and application state change events. You can register handlers for these events to respond to them.
+
+- Application behavior attributes:
+  
+    You can determine your service application behavior at boot time and after abnormal terminations by using specific attributes which you can set in the application manifest file. 
+
+The service application can be explicitly launched by a UI application. They can also be launched conditionally.
+
+The list of running service applications is available in the task switcher system application; however, no events occur if the user selects a service application from the task switcher. The main menu does not contain icons for service applications. Multiple service applications can be running simultaneously with other service and UI applications.
+
+## Application states
+
+The following figure and table describe the service application states.
+
+**Figure: Service application lifecycle**
+![Service Lifecycle](media/service_application_states.png)
+
+**Table: Service application states**
+
+| State        | Description                         |
+|--------------|-------------------------------------|
+| `READY`      | Application is launched.            |
+| `CREATED`    | Application starts the main loop.   |
+| `RUNNING`    | Application runs in the background. |
+| `TERMINATED` | Application is terminated.          |
+
+Since a service application has no UI, it does not have a pause state. Since Tizen 2.4, the service application can go into the suspended state. Basically, the service application is running in the background by its nature; so the platform does not allow running the service application unless the application has a background category defined in its manifest file. However, when the UI application that is packaged with the service application is running on the foreground, the service application is also regarded as a foreground application and it can be run without a designated background category.
```

Comment:
```suggestion
Since a service application has no UI, it does not have a pause state. Since Tizen 2.4, the service application can go into the suspended state. Basically, the service application is running in the background by its nature; so the platform does not allow running the service application unless the application has a background category defined in its manifest file. However, when the UI application that is packaged with the service application is running in the foreground, the service application is also regarded as a foreground application and it can be run without a designated background category.
```